charts/dave/charts/document-storage/templates/secret.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "daveDocumentStorage.fullname" . }}-secret
+  labels: 
+    {{- include "daveDocumentStorage.labels" . | nindent 4 }}
+stringData:
+  {{- with .Values.credentials }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)